test/lib/helpers.js

```diff
@@ -10,9 +31,18 @@ export default {
     return element;
   },
 
-  fixture: function (html) {
-    var fixture = document.getElementById('fixture');
+  fixture: function (html, tagName) {
+    if (typeof html === 'string' && typeof tagName !== 'undefined') {
```

Comment:
This entire block should be moved into the `if (typeof html === 'string')` block below.